### PR TITLE
upd(mangohud): `0.7.0` -> `0.7.1`

### PR DESCRIPTION
**Pull request by Xdavius**

### CODE REVIEW

**Comment by vigress8** [comment]
Yes, there's breaking changes in `develop`, see pacstall/pacstall#1038

File: packages/mangohud/mangohud.pacscript [hunk 1]
```diff
@@ -1,11 +1,14 @@
 name="mangohud"
 repology=("project: mangohud")
-pkgver="0.7.0"
-url="https://github.com/flightlessmango/MangoHud/releases/download/v${pkgver}/MangoHud-v${pkgver}-Source.tar.xz"
+pkgver="0.7.1"
+pkgrel="1"
```
Comment (Elsie19) [inline packages/mangohud/mangohud.pacscript:4]:
```suggestion
pkgrel="2"
```

pkgrel is implicitly 1 from the getgo.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Comment by vigress8** [comment]
Don't update manually, all the necessary changes to pacscripts will be made when `develop` is merged into `master`.

**Comment by Elsie19** [comment]
> Don't update to the new changes for now, all the necessary changes to pacscripts will be made when `develop` is merged into `master`.

The exception should be minor and major releases/hotfixes.

**Comment by Xdavius** [comment]
Does I need to add incompatible with develop to make it able to merge ?

Sorry for all the questions.

**Comment by Elsie19** [comment]
> Does I need to add incompatible with develop to make it able to merge ?
> 
> Sorry for all the questions.

No, sorry, idk what I was talking about then 😆 Let's just go back to before I said that.

**Comment by Xdavius** [comment]
> > Does I need to add incompatible with develop to make it able to merge ?
> > 
> > Sorry for all the questions.
> 
> No, sorry, idk what I was talking about then 😆 Let's just go back to before I said that.

No problem. I have reverted, I think it's all good to merge:)

File: packages/mangohud/mangohud.pacscript [hunk 2]
```diff
@@ -1,11 +1,13 @@
 name="mangohud"
 repology=("project: mangohud")
-pkgver="0.7.0"
-url="https://github.com/flightlessmango/MangoHud/releases/download/v${pkgver}/MangoHud-v${pkgver}-Source.tar.xz"
+pkgver="0.7.1"
+
 homepage='https://github.com/flightlessmango/MangoHud'
+url="${homepage}/archive/refs/tags/v${pkgver}.tar.gz"
+
```
Comment (Elsie19) [inline packages/mangohud/mangohud.pacscript:7]: I don't like this. Add the full URL for this, it kinda obscures it.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/mangohud/mangohud.pacscript (hunk 2 above)
Comment (Xdavius) [reply packages/mangohud/mangohud.pacscript]: Done

File: packages/mangohud/mangohud.pacscript [hunk 3]
```diff
@@ -1,11 +1,13 @@
 name="mangohud"
 repology=("project: mangohud")
-pkgver="0.7.0"
-url="https://github.com/flightlessmango/MangoHud/releases/download/v${pkgver}/MangoHud-v${pkgver}-Source.tar.xz"
+pkgver="0.7.1"
+
 homepage='https://github.com/flightlessmango/MangoHud'
+url="https://github.com/flightlessmango/MangoHud/archive/refs/tags/v${pkgver}.tar.gz"
+
```
Comment (Elsie19) [inline packages/mangohud/mangohud.pacscript:7]:
```suggestion
```

File: packages/mangohud/mangohud.pacscript [hunk 4]
```diff
@@ -1,11 +1,13 @@
 name="mangohud"
 repology=("project: mangohud")
-pkgver="0.7.0"
-url="https://github.com/flightlessmango/MangoHud/releases/download/v${pkgver}/MangoHud-v${pkgver}-Source.tar.xz"
+pkgver="0.7.1"
+
```
Comment (Elsie19) [inline packages/mangohud/mangohud.pacscript:4]:
```suggestion
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/mangohud/mangohud.pacscript [hunk 5]
```diff
@@ -1,11 +1,11 @@
 name="mangohud"
 repology=("project: mangohud")
-pkgver="0.7.0"
-url="https://github.com/flightlessmango/MangoHud/releases/download/v${pkgver}/MangoHud-v${pkgver}-Source.tar.xz"
+pkgver="0.7.1"
 homepage='https://github.com/flightlessmango/MangoHud'
+url="https://github.com/flightlessmango/MangoHud/archive/refs/tags/v${pkgver}.tar.gz"
```
Comment (Elsie19) [inline packages/mangohud/mangohud.pacscript:5]: Move this line back to where it was originally.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**